### DISCUSSION TITLE
[SUPPORT]  difference between *.github.com and github.com

### DISCUSSION THREAD

**Discussion by albotroz** [Cookie-AutoDelete/Cookie-AutoDelete#977] [Q&A / Support] [answered]
I've read FAQ but I don't see a explanation about the difference between url.
For example with github:

- github.com
- *.github.com

If I add github.com on whitelist what are benefits and disadvantages to *.github.com?

**Comment by kennethtran93** [comment, marked as answer]
The explanation can be found here:  https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/wiki/Documentation#enter-expression.

Simply put, `github.com` only keeps cookies and other data from that domain only, **NO** subdomains (e.g. _subdomain.github.com_).
Whereas `*.github.com` keeps cookies and other data from `github.com` as well as any/all subdomains.

The benefits/disadvantages will depend on each site and how subdomains are used.

If you trust that domain and all its subdomains, then use `*.domain.com` and all cookies under that domain and subdomains will be either kept or removed on restart.

Otherwise, or to be more secure, you'll need to keep track of which subdomain you want to keep cookies for, so you may have entries such as `keep.domain.com`, `domain.com`, `test.domain.com` on your lists, to which cookies and other data from non-listed sites, such as `second.domain.com` will be removed.

Transitioning to a discussion.
